apps/web/pages/api/cancel.ts

```diff
@@ -22,22 +23,25 @@ import { deleteScheduledEmailReminder } from "@ee/lib/workflows/reminders/emailR
 import { sendCancelledReminders } from "@ee/lib/workflows/reminders/reminderScheduler";
 import { deleteScheduledSMSReminder } from "@ee/lib/workflows/reminders/smsReminderManager";
 
-import { asStringOrNull } from "@lib/asStringOrNull";
 import { getSession } from "@lib/auth";
 import sendPayload from "@lib/webhooks/sendPayload";
 import getWebhooks from "@lib/webhooks/subscriptions";
 
 import { getTranslation } from "@server/lib/i18n";
 
+const bodySchema = z.object({
+  uid: z.string(),
+  allRemainingBookings: z.boolean().optional(),
```

Comment:
Did you test this schema? AFAIK sending `? allRemainingBookings=true` will fail since it will be a string instead of boolean. @leog